charts/kube2iam/values.yaml

```diff
@@ -102,6 +102,11 @@ resources: {}
 ##
 updateStrategy: OnDelete
 
+## Maximum number of Pods that can be unavailable during the update process
+## when using updateStrategy: RollingUpdate
+## Ref: https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#max-unavailable
+maxUnavailable: 25%
```

Comment:
if the breaking change proposed by @walkafwalka is not implemented I think this value should be optional /conditional in the template and commented out. Or it set to the default value which would be `1` for a daemonset according to https://kubernetes.io/docs/tasks/manage-daemon/update-daemon-set/. You don't want to change this value out from under people.